src/overview/components/PodsTable.tsx

```diff
@@ -0,0 +1,145 @@
+import { type FC, useState } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+import { ConsoleTimestamp } from 'src/components/ConsoleTimestamp/ConsoleTimestamp';
+import StatusIcon from 'src/components/status/StatusIcon';
+import { getResourceUrl } from 'src/modules/Providers/utils/helpers/getResourceUrl';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import type { IoK8sApiCoreV1Pod } from '@kubev2v/types';
+import { ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+import { HelperText, HelperTextItem, Pagination, Split, SplitItem } from '@patternfly/react-core';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+const getStatusLabel = (phase: string) => {
+  return (
+    <Split>
+      <SplitItem className="forklift-overview__controller-card__status-icon">
+        <StatusIcon phase={phase} />
+      </SplitItem>
+      <SplitItem>{phase}</SplitItem>
+    </Split>
+  );
+};
+
+type PodsTableProps = {
+  pods: IoK8sApiCoreV1Pod[];
+  showOwner?: boolean;
+  limit?: number;
+};
+
+export const PodsTable: FC<PodsTableProps> = ({ limit, pods, showOwner }) => {
+  const { t } = useForkliftTranslation();
+
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(10);
+
+  // Sort pods to prioritize the one starting with 'forklift-controller-'
+  const sortedPods = pods.sort((a, b) => {
+    const isForkliftControllerA = a?.metadata?.name?.startsWith('forklift-controller-') ? -1 : 0;
+    const isForkliftControllerB = b?.metadata?.name?.startsWith('forklift-controller-') ? -1 : 0;
```

Comment:
if both start with forklift-controller- ? i guess it wont continue the sort after it.. so forklift-controller-a can be after forklift-controller-b